chapter-06/inprogress/bookworm.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"sort"
+)
+
+// A Bookworm contains the list of books on a bookworm's shelf.
+type Bookworm struct {
+	Name  string `json:"name"`
+	Books []Book `json:"books"`
+}
+
+// Book describes a book on a bookworm's shelf.
+type Book struct {
+	Authors string `json:"authors"`
```

Comment:
Do we want to explicitely support plural here?

Reply:
It was to have a possible evolution but I am do not see the current value and I added the limitation to one author in the draft plan.